src/cpp/session/SessionConsoleProcess.cpp

```diff
@@ -251,6 +253,67 @@ void ConsoleProcess::commonInit()
                            module_context::activeSession().id());
 #endif
    }
+   
+   // if RStudio has been configured to use Python,
+   // place that copy of Python on the PATH
+   std::string reticulatePython = core::system::getenv("RETICULATE_PYTHON");
+   if (reticulatePython.empty())
+   {
+      Error error = r::exec::RFunction(".rs.inferReticulatePython")
+            .call(&reticulatePython);
+      
+      if (error)
+         LOG_ERROR(error);
+      
+      if (!reticulatePython.empty())
+      {
+         core::system::setenv(
+                  &*options_.environment,
+                  "RETICULATE_PYTHON",
+                  reticulatePython);
+      }
+   }
+   
+   // if we're using bash, then we'll use a custom rcfile to ensure our
+   // hooks get run when the shell is launcehd
+   if (getShellType() == TerminalShell::ShellType::PosixBash ||
+       getShellType() == TerminalShell::ShellType::GitBash)
```

Comment:
Should this also be made to work in WSLBash, too? Or is that a problem because WSL is really a separate Linux environment so the paths (and Python) it has access to can be quite different. Either way we might need some sort of warning about that (maybe a new section in the terminal support article documenting this feature's behavior and any limitations)?

Reply:
I think we could make this work for WSL as well -- let me try to test that.